packages/remotelogmetrics/events.go

```diff
@@ -63,6 +63,10 @@ type TransactionMetrics struct {
 	ScheduledTimestamp time.Time `json:"scheduledTimestamp" bson:"scheduledTimestamp"`
 	BookedTimestamp    time.Time `json:"bookedTimestamp" bson:"bookedTimestamp"`
 	ConfirmedTimestamp time.Time `json:"confirmedTimestamp" bson:"confirmedTimestamp"`
+	DeltaSolid         int64     `json:"deltaSolid"`
```

Comment:
In Kibana, there is only the host and not the nodeID, but several nodes running on the same host. Could you add the nodeID as information?